Troubleshooting-Deployment.md

```diff
@@ -43,3 +43,7 @@ role recycling. Check the logs for more information. Visual Studio provides some
 - Check the logs to see if there is an unhandled exception while initializing.
 - Make sure that the connections strings are correct.
 - Check the Azure Cloud Services troubleshooting pages for more information.
+
+## How to Check Logs
+- Use the cloud explorer in Visual Studio to navigate to the appropriate storage table or blob in the storage account. The WADLogsTable is a good starting point for looking at the logs.
+- You might only be logging errors. If you want informational logs as well, you will need to modify the configuration to set the logging severity level. You can find more information about this in the [Orleans Configuration Guide] (http://dotnet.github.io/orleans/Orleans-Configuration-Guide/).
```

Comment:
also, please avoid docs links using the full URL (since you might be hosting them somewhere else).
I believe you can just use `[Orleans Configuration Guide](Orleans-Configuration-Guide/)` and it will link correctly.